fecgov/openFEC
# Add file structure for presidential data endpoints

## Summary (required)

- Add file structure for presidential data endpoints, based on RAD Analyst (simple endpoint)

_Include a summary of proposed changes._

## How to test the changes locally

- Run this branch without `FEC_FEATURE_PRESIDENTIAL` env var set, or with it set to `FALSE` and there should be no changes
- Run this branch with `export FEC_FEATURE_PRESIDENTIAL=True` and you should see 5 presidential endpoints, based on RAD analyst. 

## Impacted areas of the application
List general components of the application that this PR will affect:

- Sets groundwork for presidential map work
- Still needs tests but we can coordinate
